partcad/src/partcad/converter.py

```
@@ -0,0 +1,169 @@
+    def _read_stl(stl_file: str) -> TopoDS_Shape:
+        """Read an STL file and return its shape."""
+        shape = TopoDS_Shape()
+        reader = StlAPI_Reader()
+        if not reader.Read(shape, stl_file):
+            raise ValueError("Failed to read the STL file")
+        return shape
+
+    @staticmethod
+    def _write_stl(shape: TopoDS_Shape, stl_file: str, linear_deflection=0.8, angular_deflection=0.6):
+        """Write a shape to an STL file."""
+        mesh = BRepMesh_IncrementalMesh(shape, linear_deflection, True, angular_deflection, True)
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)